src/test/resources/expressions/ExprRandomNumber.txt

```diff
@@ -0,0 +1,20 @@
+# Author(s):
+# 	- Skylyxx
+# Date: 2021/04/26
+
+test:
+    set {_integer} to random integer from 5 to 10
+    assert {_integer} is greater than 4 with "'random integer from 5 to 10' should be greater than 4: %{_integer}%"
```

Comment:
Needn't to put single quotes

Reply:
I though it would be more readable, but I can remove it to keep the consistency with others tests